hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/io/TestMergeHandle.java

```diff
@@ -673,74 +672,4 @@ public Map<String, HoodieRecord> getValidDeletes() {
       return validDeletes;
     }
   }
-
-  public static class CustomMerger implements HoodieRecordMerger {
```

Comment:
Both CustomMerger and CustomPayload are extracted into their own classes for reuse in testing